Sources/Kommunicate/Classes/Kommunicate.swift

```diff
@@ -357,6 +357,29 @@ open class Kommunicate: NSObject,Localizable{
             }
         })
     }
+    
+    /**
+     Updates the conversation parameters.
+     
+     - Parameters:
+     - conversation: Conversation that needs to be updated
+     */
```

Comment:
Also add `completion` param. It should be something like:
`completion: provides an update after the conversation update.`